Source/ACE/Network/GameEvent/Events/GameEventPlayerDescription.cs

```diff
@@ -50,9 +51,12 @@ private void WriteEventBody()
 
             var aceObj = Session.Player.GetAceObject() as AceCharacter;
 
-            // < 9000 to filter out our custom properties
-            var propertiesInt = aceObj.IntProperties.Where(x => x.PropertyId < 9000).ToList();
-            if (propertiesInt.Count != 0)
+            var enumValuesInt = typeof(PropertyInt).GetFields().Select(x =>
```

Comment:
These sets are actually static values that are recalculated every execution, that's sub-optimal both computationally and stylistically, I would encourage you to make them a static class property somewhere.

Reply:
Just trying to make sure I understand - the call that gets what is a server only enum is not going to change so it can be static - the comparison to the items to be written - that can change -so basically make the first part static correct?

Reply:
yup.

Reply:
ditto this.